requirements/edx/github.txt

```diff
@@ -102,5 +103,6 @@ git+https://github.com/edx/xblock-lti-consumer.git@v1.1.6#egg=lti_consumer-xbloc
 git+https://github.com/edx/edx-proctoring.git@1.2.3#egg=edx-proctoring==1.2.3
 
 # Third Party XBlocks
+git+https://github.com/doctoryes/edx-sga.git@6879a6f20b87ae828edbe4c92c86530839af04c1#egg=edx-sga==0.0
```

Comment:
I'd much rather we didn't merge this PR using the forked repo. @bmedx Have you asked MIT recently about their progress of getting the changes in the main repo?

Reply:
I think their PR is drawing to a conclusion. They contacted me with a question Friday that I just responded to: https://github.com/mitodl/edx-sga/pull/174